---
title: "Cancer-related databases"
output:
  html_document:
    toc: true
    # toc_float: true
    theme: united
    # theme: cerulean
    # number_sections: true
date: "`r Sys.Date()`"
author: "Mikhail Dozmorov"
bibliography: /Users/mdozmorov/Documents/Work/VCU_work/Mark/disease-coherence/manuscript/networks.bib
---


```{r setup, echo=FALSE, message=FALSE, warning=FALSE}
# Set up the environment
library(knitr)
opts_chunk$set(cache.path='cache/', fig.path='img/', cache=F, tidy=T, fig.keep='high', echo=F, dpi=100, warnings=F, message=F, comment=NA, warning=F, results='as.is', fig.width = 10, fig.height = 6) #out.width=700, 
library(pander)
panderOptions('table.split.table', Inf)
set.seed(1)
library(dplyr)
options(stringsAsFactors = FALSE)
```

# Survival

## GEPIA, http://gepia.cancer-pku.cn/

- Zefang Tang et al., “GEPIA: A Web Server for Cancer and Normal Gene Expression Profiling and Interactive Analyses,” Nucleic Acids Research 45, no. W1 (July 3, 2017): W98–102, https://doi.org/10.1093/nar/gkx247. - TCGA and GTEX web interface. Classical analyses - differential expression analysis, profiling plotting, correlation analysis, patient survival analysis, similar gene detection and dimensionality reduction analysis. http://gepia.cancer-pku.cn/

## KM plotter, http://kmplot.com/

Gene-centric, customizable survival analysis for breast, ovarian, lung, gastric cancers


## PRECOG, https://precog.stanford.edu

PRECOG - PREdiction of Clinical Outcomes from Genomic Profiles

Gene-centric, quick overview of survival effect of a gene across all cancers, KM plots


## PrognoScan, http://www.abren.net/PrognoScan/

Gene-centric, survival effect of a gene in cancer studies from GEO


## UALCAN, http://ualcan.path.uab.edu/

Gene-centric, tumor-normal expression, survival analusis, TCGA cancers


## R2, http://hgserver1.amc.nl/

R2: Genomics Analysis and Visualization Platform. Gene-centric, survival analysis, collection of microarray studies


## Project Betastasis, http://www.betastasis.com/

Gene-centric, survival analysis, gene expression, select cancer studies

## OncoLnc, http://www.oncolnc.org/

Gene-centric, survival analysis in any TCGA cancer

## The Human Protein Atlas, http://www.proteinatlas.org/

Uhlen, Mathias, Cheng Zhang, Sunjae Lee, Evelina Sjöstedt, Linn Fagerberg, Gholamreza Bidkhori, Rui Benfeitas, et al. “A Pathology Atlas of the Human Cancer Transcriptome.” Science (New York, N.Y.) 357, no. 6352 (August 18, 2017). doi:10.1126/science.aan2507. http://science.sciencemag.org/content/357/6352/eaan2507

Gene- and protein expression data in multiple cancer tissues, cell lines. Easy one-gene search, summary of tissue-specific expression, survival significance.

Supplementary material http://science.sciencemag.org/content/suppl/2017/08/16/357.6352.eaan2507.DC1  
- `Table S2` - summary of tissue specific expression for each gene, in normal and cancer tissues.  
- `Table S6` - summary of survival prognostic value, with a simple "favorable/unfavorable" label for each gene. Each worksheet corresponds to a different cancer.  
- `Table S8` - per-gene summary, in which cancers it is prognostic of survival.  

Rich downloadable data - tissue-specific gene expression in cancer and normal, isoform expression, protein expression. http://www.proteinatlas.org/about/download


# Misc

## OncoKB, http://oncokb.org/

- Chakravarty, Debyani, Jianjiong Gao, Sarah M. Phillips, Ritika Kundra, Hongxin Zhang, Jiaojiao Wang, Julia E. Rudolph, et al. “OncoKB: A Precision Oncology Knowledge Base.” JCO Precision Oncology 2017 (July 2017).

## TIMER, https://cistrome.shinyapps.io/timer/

- Li, Taiwen, Jingyu Fan, Binbin Wang, Nicole Traugh, Qianming Chen, Jun S. Liu, Bo Li, and X. Shirley Liu. “TIMER: A Web Server for Comprehensive Analysis of Tumor-Infiltrating Immune Cells.” Cancer Research 77, no. 21 (November 1, 2017): e108–10. https://doi.org/10.1158/0008-5472.CAN-17-0307.

TIMER is a comprehensive resource for systematical analysis of immune infiltrates across diverse cancer types. Exploring the abundances of six immune infiltrates (B cells, CD4+ T cells, CD8+ T cells, Neutrphils, Macrophages and Dendritic cells) with gene expression, survival, mutations, copy number variants and more. Six modules. Video tutorial at https://youtu.be/94v8XboCrXU

## cBioPortal, http://www.cbioportal.org/

The cBioPortal for Cancer Genomics provides visualization, analysis and download of large-scale cancer genomics data sets.


## CancerRxGene, http://www.cancerrxgene.org/

Drug-gene targets


## OncoScape, http://oncoscape.nki.nl/

Gene-centric, oncogenic scores, tumor-normal expression plots


## MDP, http://mdp.unimore.it/

MDP: Mutations and Drugs Portal. Gene-to-drug and reverse analysis


## CTRP, https://portals.broadinstitute.org/ctrp/

The Cancer Therapeutics Response Portal (CTRP) links genetic, lineage, and other cellular features of cancer cell lines to small-molecule sensitivity with the goal of accelerating discovery of patient-matched cancer therapeutics.


## MEXPRESS, http://mexpress.be/

Gene-centric methylation and correlation with clinical parameters

## PCAGW, http://docs.icgc.org/pcawg/

The PCAWG study is an international collaboration to identify common patterns of mutation in more than 2,800 cancer whole genomes from the International Cancer Genome Consortium. The project produced large amount data with many types including simple somatic mutations (SNVs, MNVs and small INDELs), large-scale somatic structural variations, copy number alterations, germline variations, RNA expression profiles, gene fusions, and phenotypic annotations etc. PCAWG data have been imported, processed and made available in the following four major online resources for download and exploration by the cancer researchers worldwide.

- Goldman, Mary, Junjun Zhang, Nuno A. Fonseca, Qian Xiang, Brian Craft, Elena Piñeiro, Brian O’Connor, et al. “Online Resources for PCAWG Data Exploration, Visualization, and Discovery.” BioRxiv, October 18, 2017. https://doi.org/10.1101/163907. https://www.biorxiv.org/content/early/2017/10/18/163907

## NCI Genomics Data Commons API

- Shane Wilson, Michael Fitzsimons, Martin Ferguson, Allison Heath, Mark Jensen, Josh Miller, Mark W. Murphy, James Porter, Himanso Sahni, Louis Staudt, Yajing Tang, Zhining Wang, Christine Yu, Junjun Zhang, Vincent Ferretti and Robert L. Grossman. "Developing Cancer Informatics Applications and Tools Using the NCI Genomic Data Commons API." DOI: 10.1158/0008-5472.CAN-17-0598 Published November 2017 http://cancerres.aacrjournals.org/content/77/21/e15

https://docs.gdc.cancer.gov/API/Users_Guide/Getting_Started/ - docs

https://github.com/Bioconductor/GenomicDataCommons - R package


# Links and References

- Zhang, Zhuo, Hao Li, Shuai Jiang, Ruijiang Li, Wanying Li, Hebing Chen, and Xiaochen Bo. “A Survey and Evaluation of Web-Based Tools/Databases for Variant Analysis of TCGA Data.” Briefings in Bioinformatics, March 29, 2018. https://doi.org/10.1093/bib/bby023. - The most comprehensive review of TCGA-related tools. Table 3 - List of Web servers and databases. 

- The Pan-Cancer analysis by TCGA consortium, all papers. https://www.cell.com/pb-assets/consortium/pancanceratlas/pancani3/index.html

- TCGA mutation data, MAF files, open and controlled access. https://gdc.cancer.gov/about-data/publications/mc3-2017

